src/pint/models/__init__.py

```diff
@@ -51,13 +57,3 @@
 from pint.models.troposphere_delay import TroposphereDelay
 from pint.models.wave import Wave
 from pint.models.wavex import WaveX
-
-# Define a standard basic model
-StandardTimingModel = TimingModel(
-    "StandardTimingModel",
-    [AstrometryEquatorial(), Spindown(), DispersionDM(), SolarSystemShapiro()],
-)
-# BTTimingModel = generate_timing_model("BTTimingModel",
-#         (Astrometry, Spindown, Dispersion, SolarSystemShapiro, BT))
-# DDTimingModel = generate_timing_model("DDTimingModel",
-#         (Astrometry, Spindown, Dispersion, SolarSystemShapiro, DD))
```

Comment:
This looks unnecessary. None of the tests depend on it, at least. It looks like vestigial lines of an early design approach that changed afterward.